### DISCUSSION TITLE
Can't get minestom to start

### DISCUSSION THREAD

**Discussion by Flexiik** [Minestom/Minestom#1557] [Help]
I have this error and have no idea how to fix it, tried multiple versions, but it quite not seems to run well. I am using groovy for gradle and JDK 17
![image](https://user-images.githubusercontent.com/57681244/206303670-997054ae-33bc-4ebd-95cd-809f59cf9850.png)

**Comment by DevBlokJochem** [comment]
Hey, do you already have a main class? Can you send the main class if you have it?

**Comment by Flexiik** [comment]
Right here :)
![image](https://user-images.githubusercontent.com/57681244/206304700-2255a264-e478-4651-9db6-9d78f27b067f.png)

**Reply by Flexiik** [reply]
Just wanted to start a simple server on 1.19.3

**Reply by GreatWyrm** [reply]
I'd also recommend posting your build.gradle file, and a couple of things I noticed

1. ~~You don't need to add MinestomDataGenerator as a dependency~~, not sure what's going on there, but I don't think you explicitly listed that as a dependency.
2. Your main method isn't fully complete (you need to create an instance and register an event to spawn players there), more details [here](https://wiki.minestom.net/setup/your-first-server).

Additionally, 1.19.3 isn't quite ready for Minestom, but I imagine it will be completed in a couple of days

**Reply by Flexiik** [reply]
It can be 1.19.x (I'll be glad if it works :D)
```
plugins {
    id 'java'
    id "com.github.johnrengelman.shadow" version "6.1.0"
}

java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(17)
    }
}

compileJava.options.encoding = 'UTF-8'

tasks.withType(JavaCompile) {
    options.encoding = 'UTF-8'
}

group 'cz.flexiik'
version '0.1-ALPHA'

repositories {
    mavenCentral()
    maven { url 'https://repo.spongepowered.org/maven' }
    maven { url 'https://jitpack.io' }
}

dependencies {
    implementation 'com.github.Minestom:Minestom:-SNAPSHOT'
}

test {
    useJUnitPlatform()
}

jar {
    manifest {
        attributes 'Main-Class': 'fr.themode.minestom.MinestomTest',
                "Multi-Release": true
    }
}
```
![image](https://user-images.githubusercontent.com/57681244/206306052-75eb1d88-99b4-445d-9d16-d94e946fea7d.png)
![image](https://user-images.githubusercontent.com/57681244/206306101-528db840-a8ae-4089-85a1-489beea2177c.png)

**Reply by ShiftSad** [reply]
Same for me, could not find MinstomDataGenerator also.

**Reply by GreatWyrm** [reply]
Okay, a few things about your build.gradle

1. I'd upgrade `com.github.johnrengelman.shadow` to the latest version (7.1.2)
2. Don't use -SNAPSHOT to get a Minestom version, look at the [Jitpack](https://jitpack.io/#Minestom/Minestom) to get the latest version (com.github.Minestom.Minestom:Minestom:7361bf0825)
3. Change the path of the manifest Main class, it's not `fr.themode.minestom.MinestomTest`.

**Reply by FluxCapacitor2** [reply]
Can you try again now? It seems like Jitpack rebuilt it today for some unknown reason
(build log: https://jitpack.io/com/github/Minestom/MinestomDataGenerator/1c1921cd41/build.log)

**Reply by Flexiik** [reply]
Thank you so much, it works now <3
